apps/prairielearn/src/ee/lib/selectAndAuthz.ts

```diff
@@ -19,12 +19,16 @@ export async function selectAndAuthzInstitutionAsAdmin({
     { institution_id, user_id },
     z.object({
       institution: InstitutionSchema,
-      administrator: AdministratorSchema,
+      administrator: AdministratorSchema.nullable(),
+      institution_administrator: AdministratorSchema.nullable(),
     }),
   );
 
-  // TODO: support institution-level administrators once they exist.
-  if (result == null || !result.administrator || !access_as_administrator) {
+  if (
+    result == null ||
+    (!result.administrator && !result.institution_administrator) ||
```

Comment:
Should we implement support for the "access as administrator" setting for institution admins?

Reply:
It feels like it would be a bit of a mess from a UI perspective, because the home page isn't rendered in the context of any given institution so the menu to turn on/off institution admin access would need to switch depending on which page you are viewing?

Honestly, I'd be fine with ripping out the "access as administrator" functionality altogether (that is, admins are always admins).

Reply:
True, that would be tricky to build UI for. I'm happy to avoid this for institution admins, but I also don't feel like there's a huge need to remove that functionality from global admins either.